src/deposit/dto/create-deposit.dto.ts

```diff
@@ -1 +1,5 @@
-export class CreateDepositDto {}
+export class CreateDepositDto {
+  userId: string;
```

Comment:
tutaj trzeba to połączyć z UserInterface, wtedy zmieniłabym tez nazwę po prostu na user